aws_lambda_builders/workflows/dotnet_clipackage/dotnetcli.py

```diff
@@ -47,9 +48,17 @@ def run(self, args, cwd=None):
 
         LOG.debug("executing dotnet: %s", invoke_dotnet)
 
+        if tool_dir is None:
+            env = None
+        else:
+            env = SubprocessDotnetCLI._merge(
```

Comment:
I'm far from a Python expert, so if I've made a hash of this, please do let me know.

Reply:
We have dropped support for running Lambda builder in Python 2.
Let's directly use `env.copy().update(...)` instead.